Realm/RLMAccessor.mm

```diff
@@ -576,19 +576,25 @@ static RLMAccessorCode accessorCodeForType(char objcTypeCode, RLMPropertyType rl
     }
 }
 
+static void RLMReplaceShouldIncludeInDefaultSchemaMethod(Class cls, bool shouldInclude) {
+    Class metaClass = objc_getMetaClass(class_getName(cls));
+    IMP imp = imp_implementationWithBlock(^(Class){ return shouldInclude; });
+    class_replaceMethod(metaClass, @selector(shouldIncludeInDefaultSchema), imp, "b@:");
+}
+
 // implement the class method className on accessors to return the className of the
 // base object
 void RLMReplaceClassNameMethod(Class accessorClass, NSString *className) {
     Class metaClass = objc_getMetaClass(class_getName(accessorClass));
-    IMP imp = imp_implementationWithBlock(^{ return className; });
-    class_addMethod(metaClass, @selector(className), imp, "@:");
```

Comment:
addMethod is the correct function for all of these functions. replaceMethod is significantly slower and is only needed if the class already has a definition of that method which needs to be replaced (as opposed to overriding a method from a parent class). RLMReplaceSharedSchemaMethod uses replaceMethod because opening a Realm file with a different column ordering used to modify the existing accessor classes rather than creating new ones, and it was never switched to addMethod when we fixed that.